browser.js

```diff
@@ -26,13 +32,30 @@ function randomBytes (size, cb) {
   }
 
   // XXX: phantomjs doesn't like a buffer being passed here
-  var bytes = Buffer.from(rawBytes.buffer)
+  return Buffer.from(rawBytes.buffer)
+}
+
+function randomBytes (size, cb) {
+  if (size > MAX_UINT32) {
+    throw new Error('Requested too many random bytes - the maximum allowed are ' + MAX_UINT32)
+  }
+
+  var bytes = Buffer.alloc(0)
+
+  if (size > MAX_BYTES) {
+    bytes = randomBytes(size - MAX_BYTES)
```

Comment:
we should probably not do this recursively instead we should probably make all the sub buffers and concat them once, or create a single ArrayBuffer and pass views of MAX_BYTES or less so you don't need to even do a concat

Reply:
As you prefer.